### PR TITLE
[67402] Adding authentication time to Sign in Service and SSOe logs

### PR DESCRIPTION
**Pull request by bosawt**
## Summary

- This PR adds a new field to completed sign in service and SSOe authentications completion logs: `authentication_time`
- On Sign in Service, this is in the `callback` success log
- On SSOe, this is in the `SSOe: SAML Response` log

## Related issue(s)
- department-of-veterans-affairs/va.gov-team#67402

## Testing done

- Authenticated with Sign in Service and SSOe, confirmed that new logs are present with `authentication_time` field

## What areas of the site does it impact?
Authentication Metrics

## Acceptance criteria

- [ ] Authenticate with SSOe, confirm `authentication_time` is present in `SSOe: SAML Response ` log
- [ ] Authenticate with Sign in Service, confirm `[SignInController] callback` log has field, `authentication_time` on it

### CODE REVIEW

File: .github/CODEOWNERS
Comment (bosawt) [inline .github/CODEOWNERS:130]: I'm switching `controllers/v1` over to backend review, as there are both lighthouse and non-lighthouse stuff in there

**Review by RachalCassity — APPROVED** [review]

**Review by dickdavis — APPROVED** [review]
LGTM. Man, such a difference in time between SiS and SSOe. 😳

SignInService log:

```
[SignInService] [V0::SignInController] callback -- { :type => "logingov", :client_id => "vaweb", :ial => 2, :acr => "ial2", :icn => "1008709396V637156", :uuid => "381ce98f-d604-43c9-9512-2088d0983e2d", :authentication_time => 5 }
```

SSOe log:

```
SSOe: SAML Response => {"id"=>"_9a599d71-6e81-4196-b911-70f7be4de7a3", "authn"=>"http://idmanagement.gov/ns/assurance/loa/1/vets", "type"=>"idme", "tran
saction_id"=>"80d60879-aee1-43d7-a8dc-32532d6f54c0", "authentication_time"=>189}
```

**Comment by bosawt** [comment]
@department-of-veterans-affairs/lighthouse-banana-peels Can I get someone from this group to review this? This group was only part of this PR because the CODEOWNERS had this group for all v1 controllers, otherwise it doesn't really relate, but I can't merge without an approval

**Review by dickdavis — APPROVED** [review]

**Review by RachalCassity — APPROVED** [review]

**Pull request merged**
